### PR TITLE
[WIP] expand Profile()

### PR DESCRIPTION
**Pull request by kingrpaul**

### CODE REVIEW

**Comment by kingrpaul** [comment]
> Can you make a pull request and comment on that instead?
Yes.

**Comment by kingrpaul** [comment]
> I found the issue you're referring to. Could you merge master in to your branch: ```bash git pull git merge origin/master [resolve any conflicts in vscode and commit if need be] git push ``` That particular problem you're hitting is an issue with the Windows image on Azure pipelines. I'll need to troubleshoot it in the near future. Master branch for now has disabled testing of Windows. Cheers, Simon

Thanks. Scoping that out, now.

**Comment by kingrpaul** [comment]
> > I found the issue you're referring to. Could you merge master in to your branch: `bash git pull git merge origin/master [resolve any conflicts in vscode and commit if need be] git push ` That particular problem you're hitting is an issue with the Windows image on Azure pipelines. I'll need to troubleshoot it in the near future. Master branch for now has disabled testing of Windows. Cheers, Simon
> 
> Thanks. Scoping that out, now.

Thank you @SimonBiggs. Got the green check. That sorted it out.

**Comment by SimonBiggs** [comment]
Awesome, glad that worked :)

**Comment by SimonBiggs** [comment]
@kingrpaul did you want this reviewed and merged?

File: src/pymedphys/_labs/paulking/profile.py [hunk 1]
```diff
@@ -36,17 +36,17 @@
...
 import PIL
 
-from ...libutils import get_imports
-
-# from .._level1.coreobjects import _PyMedPhysBase
-IMPORTS = get_imports(globals())
+try:
+    assert "dosepro" not in __file__
+    from ...libutils import get_imports
+    IMPORTS = get_imports(globals())
```
Comment (SimonBiggs) [inline src/pymedphys/_labs/paulking/profile.py]: It looks like this import will always fail. May as well delete these lines of code here.

Comment (SimonBiggs) [reply src/pymedphys/_labs/paulking/profile.py]: @kingrpaul I'm happy to merge this by the way, just figured it's worth letting you know if I see glaring issues...

Let me know if you want me to merge.

Comment (kingrpaul) [reply src/pymedphys/_labs/paulking/profile.py]: @SimonBiggs much appreciated. I have things I want to do with this but it's not ready for merge yet.

Comment (SimonBiggs) [reply src/pymedphys/_labs/paulking/profile.py]: Kk :)

**Comment by SimonBiggs** [comment]
@kingrpaul there's been some big changes to the repo organisation, I'll need to merge master into this branch and sort it out for you a little bit. I should get a chance tonight at some point...

**Comment by SimonBiggs** [comment]
@kingrpaul check out the work in progress web application:

https://app.pymedphys.com

I need to fix the freezing, working on that. But essentially this opens up the ability to make a large range of file processing scripts accessible to users via a no-installation web interface.

**Comment by SimonBiggs** [comment]
@kingrpaul any progress on this one? Best to not leave a pull request hanging. What needs to happen to get this one merged?

**Comment by SimonBiggs** [comment]
Will close this pull request for now. We can certainly open it back up again if you wanted to jump back in.

**Pull request closed without merging**